pins/tests/_databackend/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Technically complying with MIT requires distributing the license

Reply:
Actually this probably need to be moved out of `tests` because I'm not sure if that gets included in the wheel build (I think it doesn't).

Reply:
The tests directory does get included in the wheel build! I'm not entirely sure where this file should be, but it might be easier to not vendor in this package, if possible 😄